js/scripts.js

```diff
@@ -1420,6 +1408,25 @@ var plugin_formcreator = new function() {
 
       return true;
    };
+
+   this.addEmptyCondition = function (target) {
+      var form     = target.closest('form[data-itemtype]');
+      var itemtype = form.getAttribute('data-itemtype');
+      var id       = form.getAttribute('data-id') || null;
+      var data     = $(form).serializeArray();
+      data.push({name: 'itemtype', value: itemtype});
+      data.push({name: 'items_id', value: id});
+      $.post({
+         url: formcreatorRootDoc + '/ajax/condition.php',
+         data: data,
+      }).done(function (data) {
+         target.closest('div.row').after(document.createRange().createContextualFragment(data));
+      });
+   }
+
+   this.removeNextCondition = function (target) {
+      target.closest('div.row').remove();
+   }
```

Comment:
removed use of FormData object, while movingthe function in the encapsulation class for  the plugin